gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
Ah. My original question should have been "does anyone ever build on Windows?".

Thanks, Max.